components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/model/SAMLSSOServiceProviderDO.java

```diff
@@ -174,6 +176,26 @@ public void setIssuer(String issuer) {
         }
     }
 
+    /**
+     * Get qualifier of the issuer.
+     *
+     * @return Issuer Qualifier
+     */
+    public String getIssuerQualifier() {
+        return issuerQualifier;
```

Comment:
Add a new line after the method signature.

Reply:
Addressed in https://github.com/wso2/carbon-identity-framework/pull/3100